neuroanalysis/miesnwb.py

```diff
@@ -404,7 +405,7 @@ def __getstate__(self):
 class MiesTestPulse(PatchClampTestPulse):
     def __init__(self, entry, rec):
         chan = rec.device_id
-        clamp_mode = 'vc' if np.isnan(entry['TP Baseline Vm']) else 'ic'
+        clamp_mode = 'vc' if np.isnan(entry['TP Baseline Vm'][chan]) else 'ic'
```

Comment:
How about removing this line and adding to 415: 

    clamp_mode = 'vc' if np.isnan(self._nb_entry['TP Baseline Vm']) else 'ic'

..because I think sometimes there will not be an array to index from, and 410-413 takes care of that possibility.

Reply:
Yes, that is better.